decisions/0015-env-triad-terms.md

```diff
@@ -1,52 +1,124 @@
+- This ADR will be updated as environment-specific queries are created.
+- The logic described above is intended to minimize cherry-picking of values for the sets, but individual squad members
+  will be given the opportunity to add their own `Y` or `N` votes for each value in the tables. It may be possible to
+  feed the insights from these votes back into reusable logic, but that might require complex OAK queries over
+  inconsistent annotations, or grouping of values within a semantic embedding space. For example, there may be evidence
+  that [farm [ENVO:00000078]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000078),
+  [agricultural field [ENVO:00000114]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000114)
+  and [banana plantation [ENVO:00000161]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000161)
+  should all be included in the `env_local_scale` enumeration for Soil samples. It may be more expeditious for
+  a human to make the judgment call to include farm and exclude the others, compared to the effort required to automate
+  that. Visualizations will be provided to illustrate these cases. See the end of this ADR.
+- As necessary, we can request that EnvO add classes in support of this work, or that more (and more consistent) axioms
+  are added to existing classes.
 
-To ensure interoperability and consistancy, the NMDC General Query will be ammended for each environmental sample type.
-- To satisfy NMDC needs, we will start here with soil.
-- This ADR will be updated as environment specific queries are created.
-- No environment specific query should be "cherry picked". Rather any filtering should be accomplished as a general query. (Example, will not remove a specific term by ID, but rather identify what is the rule that can be created to fit a overall need.)
+## NMDC General Queries for Environmental Triad Value Sets
 
-NMDC General Query
-- `env_broad_scale` will be composed of terms that branch from biome [ENVO:00000428]
-  - We will then evaluate what is lost when ecosystem is excluded and determine what needs “re-added” or requested of envo
-- `env_local_scale` will be composed of terms that branch from material entity [BFO:0000040], excluding biome [ENVO:00000428] & environmental material [ENVO:00010483]
-  - Additional narrowing will occur to remove specific branches. 
-- `env_medium` will be composed of terms that branch from environmental material [ENVO:00010483]
+- `env_broad_scale` will consist of subclasses
+  of [biome [ENVO:00000428]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000428),
+  with the exception of host-associated samples, including plant-associated samples.
```

Comment:
```suggestion
  with the exception of host-associated samples, including plant-associated samples.
```

What? exception of host, ok that makes snes
but what plant-associated? IDK what you're saying here